### ISSUE TITLE
CreateYobicode method returned "error":"invalid method".

### ISSUE THREAD

**Issue by kamilongus** [siredukov/YobitPython#1]
Сам запрос: 
`method=CreateYobicode&currency=DOGE&amount=0.0001&nonce=11`

Ответ:
```
{
    "success": 0,
    "error": "invalid method"
}
```
Алиас метода - CreateCoupon также не работает.

**Comment by mikeedi** [comment]
Да, действительно выдает invalid method, даже если использовать альтернативный метод CreateCoupon. Такая же ошибка в RedeemYobicode. Написал в поддержку.

**Comment by ktnik** [comment]
с RedeemYobicode таже проблема, неделю мучаюсь с разными языками везде ошибка

**Comment by mikeedi** [comment]
На тикет в поддержке никто не отвечает. Как я понял из тг чатов йобита баги с api в порядке вещей. Еще сегодня заметил проблемы с TradeHistory (возвращает 1 на любой вызов).

**Comment by kamilongus** [comment]
Не могли они вообще убрать эти методы из API? Выглядит именно так.

**Comment by mikeedi** [comment]
Возможно.
